static/js/common/fxa-login.js

```diff
@@ -35,23 +31,24 @@
     function fxaLogin(opts) {
         opts = opts || {};
         var authConfig = {
+            action:  'signin',
+            client_id: config.clientId,
             email: opts.email || config.email,
             state: config.state + ':' + urlsafe(btoa(nextPath())),
             redirectUri: config.redirectUrl,
             scope: config.scope,
         };
-        if (opts.signUp) {
-            console.log('[FxA] Starting register');
-            return fxaClient.auth.signUp(authConfig);
-        } else {
-            console.log('[FxA] Starting login');
-            return fxaClient.auth.signIn(authConfig);
+        if (opts.migration) {
+            authConfig.migration = 'amo';
         }
+        var url = config.oauthHost + '/authorization?' + $.param(authConfig);
+        console.log('[FxA] Starting login', url);
```

Comment:
We're probably never going to see it but if login stops working it will be nice to know if they made it this far.